operations/helm/tests/faro-ingress/grafana-agent/templates/ingress.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 # Source: grafana-agent/templates/ingress.yaml
-apiVersion: networking.k8s.io/v1
```

Comment:
I honestly do not know why this changed. Tool version differences?

Reply:
Try `USE_CONTAINER=1 make generate-helm-tests`